package.json

```diff
@@ -22,7 +22,8 @@
         "lint": "concurrently \"cd shared && npm run lint\" \"cd frontend && npm run lint\" \"cd backend && npm run lint\"",
         "lint:fix": "concurrently \"cd shared && npm run lint:fix\" \"cd frontend && npm run lint:fix\" \"cd backend && npm run lint:fix\"",
         "test:concurrently": "concurrently \"cd shared && npm run test\" \"cd frontend && npm run test\" \"cd backend && npm run test\"",
-        "test": "cd shared && npm run test && cd ../frontend && npm run test && cd ../backend && npm run test",
+        "test": "(mkdir test-scenarios || cd .) && cd test-scenarios && (git clone https://github.com/hpi-sam/digital-fuesim-manv-public-test-scenarios.git || cd .) && cd digital-fuesim-manv-public-test-scenarios && git pull && cd ../../shared && npm run test && cd ../frontend && npm run test && cd ../backend && npm run test",
```

Comment:
Maybe consider [Git Submodules](https://git-scm.com/book/en/v2/Git-Tools-Submodules) for this.

Reply:
I will thanks for the hint